src/gatling/simulations/uk/gov/hmcts/reform/sscs/performance/utils/Environment.scala

```diff
@@ -2,15 +2,17 @@ package uk.gov.hmcts.reform.sscs.performance.utils
 
 object Environment {
 
+  val env = "aat"
```

Comment:
I like this, might be worth adding a log message at the start of your simulation to output the environment it's running in? Your call.